src/traces/scattersmith/format_labels.js

```diff
@@ -0,0 +1,35 @@
+'use strict';
+
+var Lib = require('../../lib');
+var Axes = require('../../plots/cartesian/axes');
+
+module.exports = function formatLabels(cdi, trace, fullLayout) {
+    var labels = {};
+
+    var subplot = fullLayout[trace.subplot]._subplot;
+    var radialAxis;
+    var angularAxis;
+
+    // for scatterpolargl texttemplate, _subplot is NOT defined, this takes part during the convert step
```

Comment:
No `scatterpolargl` or `scattersmithgl`. So we could simplify this.

Reply:
43a3be8db7ee003ed483ec79170c499f7c29e874

Reply:
You could remove calls to `redrawReglTraces` as well.